tests/Rules/Function/IncludeFunction/IncludeFunctionRuleTest.fixed.twig

```diff
@@ -0,0 +1,14 @@
+{{ include('template.html' ) }}
+{{ include(['template_a.html', 'template_b.html']   , {'foo': 'bar'}  , ignore_missing = true, with_context = false) }}
+{{ include('template.html'  , with_context = false) }}
+{{ include('template.html' , vars ) }}
+{{ include('template.html' , vars  , with_context = false) }}
+{{ include('template.html' , {'foo': 'bar'} ) }}
+{{ include('template.html' , {'foo': 'bar'}  , with_context = false) }}
```

Comment:
You started to handled unnecessary whitespace with your 
```
if ($this->isTokenMatching($tokens[$tokenPosition + 1], Token::WHITESPACE_TOKENS)) {
            $fixer->replaceToken($tokenPosition + 1, '');
        }
```
check, but we still have unnecessary one here, before the `,`.

Imho, we should either
- complicate the rule to handle correctly ALL the whitespace
- or handle none

Reply:
I'd say none, even if it's a bit weird-looking in the "fixed.twig" files... (i went back and forth on this when i coded)

Do you have any preference ?

Reply:
(removed from now, tell me if you want otherwise)

Reply:
None seems the simpler.

We can still iterate on it later if someone complains about it.

Our recommendation would to use both the include fixer with all the other spacing fixer